Source/monster.h

```diff
@@ -316,6 +316,11 @@ struct Monster { // note: missing field _mAFNum
 		return *type().data;
 	}
 
+	bool IsAlive() const
+	{
+		return hitPoints > 0;

```

Comment:
Should we ignore the hp fractions for monster and players or allow monsters and player with hp "0" (and some fraction) to be alive? 🤔 

```suggestion
		return (hitPoints >> 6) > 0;
```